### DISCUSSION TITLE
dynamic url w/require

### DISCUSSION THREAD

**Discussion by karzol73** [parcel-bundler/parcel#6919] [Q&A] [answered]
Hi!

I would like to change the video src dynamically in js like this:

let selectedVideo = 1;
let source = `../assets/video/${selectedVideo}.mp4`;
video.src = require(source);

but it does not work while the following is running correctly:

video.src = require("../assets/video/1.mp4");

What is the correct method to use src dynamiclly with Parcel? 

Thanks for all!

**Comment by mischnic** [comment, marked as answer]
You could use globs: https://v2.parceljs.org/blog/rc0/#glob-resolver-plugin

**Reply by karzol73** [reply]
Thank you!
